### DISCUSSION TITLE
Default  screen for floating windows  in tablet mode

### DISCUSSION THREAD

**Discussion by sir86** [Archfx/duo-de#96] [Ideas]
I use Floating windows in tablet mode and apps always open on left screen  by default . its possible to make them open on right screen by default ?

**Comment by Archfx** [comment]
It possible to implement this. If someone can volunteer, here is [patch](https://github.com/Archfx/duo-de/blob/main-15/patches/duo/platform_packages_apps_Trebuchet/0005-ARCHFX-DUO-DE-Launch-freeform-apps.patch) related to floating windows. 

```shell
Rect rect = new Rect(
+            100,
+            120,
+            1244,
+            1600);
```

This logic determines the location of the floating window on the screen which is currently set for the left screen. Ideal way would be to add user options to `/res/xml/launcher_preferences.xml` to select left, right or automatic and change the above values accordingly.

**Comment by sir86** [comment]
Would be cool if someone could add it . For me it has more sense that way cuz when u open Duo then u dont have to open it fully  and still use right screen. Now u can still do it but u have to move any opened app to right side manualy but if u close the app , u have to repeat it cuz it will open back on left screen. The only way is to minimalize window so it will remember its position but i would prefer to have it default on right side. It was made smiller way on stock 12L. u could open duo like phone with cover , use right screen and close it. I think its cool feature.

**Comment by sir86** [comment]
there is actually option like that in Taskbar.

**Comment by Archfx** [comment]
I was working on a better solution for this actually, need to find a free time to implement it. lets see

**Comment by Archfx** [comment]
I hope the latest release addresses your concern!
